rasa/core/events/__init__.py

```diff
@@ -1034,10 +1060,11 @@ def __eq__(self, other) -> bool:
         if not isinstance(other, ActionExecuted):
             return False
         else:
-            return (
-                self.action_name == other.action_name
-                and self.e2e_text == other.e2e_text
-            )
+            equal = self.action_name == other.action_name
+            if hasattr(self, "action_text") and hasattr(other, "action_text"):
```

Comment:
@tmbo Getting pickled events these old seems very unlikely at this point (we only used it for the `RedisTrackerStore` and I remember Rick changing that a long time ago). Can we safely dump the `hasattr` usage in Rasa Open Source with the new major?